processing/app/fixtures.py

```diff
@@ -1,23 +1,40 @@
+import json
 import uuid
 
 from app.core import config
 from app.db.sessions import IntegrityError
 from app.db.sessions import get_async_engine
 from app.db.sessions import get_async_session_factory
+from app.models.survey import Survey
 from app.models.survey_answer import SurveyAnswer
 from app.models.task import Task
 
 
 async def load_initial_data():
     cfg = config.PostgresSettings.from_env()
+    example_schema = {
+        "title": "A registration form",
+        "description": "",
+        "type": "object",
+        "required": [
+            "firstName",
+        ],
+        "properties": {"firstName": {"type": "string", "title": "First name", "default": "Chuck"}},
+    }
+    survey = Survey(
```

Comment:
we should not init survey data in processing (the lets move this into db directly there is an way in postgres docker image)

Reply:
So we are fine with creating `SurveyAnswers` that are Survey specific but not fine with Survey? 😅